packages/core/docs/.vuepress/styles/index.styl

```
@@ -25,3 +25,22 @@ a code {
   color: $accentColor !important;
   font-weight: bold;
 }
+
+.image-shadow {
+  display: block;
+  margin-top: 20pt;
+  margin-bottom: 20pt;
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 0px 0px 5px 6px rgba(0, 0, 0, 0.30);
```

```suggestion
  box-shadow: 0px 10px 15px 6px rgba(0, 0, 0, 0.30);
```
Maybe sth like that? At least looks a bit better to me :D

---

looks better, thx